rero_mef/utils.py

```diff
@@ -387,30 +392,31 @@ def oai_save_records_from_dates(name, file_name, sickle, oai_item_iterator,
             }
             if access_token:
                 params['accessToken'] = access_token
-            params.update(dates)
+            params |= dates
```

Comment:
This operation on dict are only available with `>=python3.9`. If you want MEF project stays compatible with older project, continue to use `.update()` method.
We had the same problem with `rero-invenio-base` project, we choose to keep old method to not force children project to have python3.9.


(Sourcery plugin proposition I think ;-) )

Reply:
No problem here MEF is python version 3.9